src/openzaak/components/zaken/tests/test_zaken.py

```diff
@@ -574,3 +577,131 @@ def test_zaak_archiefactiedatum_afleidingswijze_ingangsdatum_besluit(self):
         self.assertEqual(
             zaak.archiefactiedatum, date(2030, 5, 3)  # 2020-05-03 + 10 years
         )
+
+
+@tag("external-urls")
+class ZaakCreateExternalURLsTests(JWTAuthMixin, APITestCase):
+    heeft_alle_autorisaties = True
+    list_url = get_operation_url("zaak_create")
+
+    def test_create_external_zaaktype(self):
+        catalogus = "https://externe.catalogus.nl/api/v1/catalogussen/1c8e36be-338c-4c07-ac5e-1adf55bec04a"
+        zaaktype = "https://externe.catalogus.nl/api/v1/zaaktypen/b71f72ef-198d-44d8-af64-ae1932df830a"
+
+        with requests_mock.Mocker(real_http=True) as m:
+            m.register_uri(
+                "GET", zaaktype, json=get_zaaktype_response(catalogus, zaaktype),
+            )
+            m.register_uri(
+                "GET",
+                catalogus,
+                json={
+                    "url": catalogus,
```

Comment:
probably better to make a `get_catalogus_response` as well?

Reply:
`get_catalogus_response` is added to the utils